client/my-sites/theme/controller.jsx

```diff
@@ -60,7 +60,12 @@ export function details( context, next ) {
 	}
 
 	context.primary = (
-		<ThemeSheetComponent id={ slug } section={ section } pathName={ context.pathname } />
+		<ThemeSheetComponent
+			id={ slug }
+			section={ section }
+			themeSlug={ slug }
```

Comment:
We already have the `slug`.

It's called...

`id`...

Quite counterintuitive, but I'd avoid passing down the same props twice. 🙂

Reply:
Ha, @Copons beat me to it 🙃 Otherwise, works as expected 👍

Reply:
you were too fast!

I was actually working the other way around, removing `id` because it's use inside the `ThemeSheetComponent` smells really bad. But that'll require another PR. So I removed `themeSlug` in c0786d340d535d1237946fb21ac436e87577c7e1.